keyboard.html

```diff
@@ -36,25 +37,28 @@
     const allBlackKeysArray = Array.from(document.getElementsByClassName('black-key'))
     
     function addKeys(numKeys) {
-      const whiteKeysPerOctave = 6
+      const keysPerOctave = 11
       let i = 0;
-      let whiteKeyCount = 0;
+      let keyCount = 0;
       while(i < numKeys) {
         // create a new rectangle div element
         const newDiv = document.createElement("div");
         newDiv.classList.add('white-key')
+        newDiv.setAttribute('key', `${keyNames[keyCount]}`)
```

Comment:
maybe making `addWhiteKey` its own function that increments `keyCount` and groups all the logic together will help clarify why its being incremented 2x